tools/linter_lib/custom_check.py

```diff
@@ -406,9 +406,11 @@ def custom_check_file(fn, identifier, rules, color, skip_rules=None, max_length=
         {'pattern': "{{ _(.+) }}[\.\?!]",
          'description': "Period should be part of the translatable string."},
     ]
-    json_rules = []  # type: RuleList # fix newlines at ends of files
-    # It is okay that json_rules is empty, because the empty list
-    # ensures we'll still check JSON files for whitespace.
+    json_rules = [
+        {'pattern': '\s+$',
+         'strip': '\n',
+         'description': 'Fix trailing whitespace'},
+    ]
```

Comment:
The clearly better plan there would be to make `trailing_whitespace_rule` its own variable and use that in both places.